Makefile

```diff
@@ -63,6 +63,12 @@ check-static: ## Run static code analysis checks
 check-static: tools/bin/golangci-lint
 	GO111MODULE=on CGO_ENABLED=0 tools/bin/golangci-lint run -v $$($(PACKAGE_DIRECTORIES)) --config .golangci.yml
 
+.PHONY: check-ddl
+check-ddl: tools/bin/golangci-lint
+	GO111MODULE=on CGO_ENABLED=1 ./tools/bin/golangci-lint run -v pkg/ddl \
+		--config pkg/ddl/.golangci.yml \
+		--new-from-merge-base=$$(git remote -v | awk '/pingcap\/tidb.git/ {print $$1; exit}')/master
```

Comment:
When cherry-pick, the merge base is not master? When checkout new release-branches, please remember to update it.